<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PipFileSelectionDialog</class>
 <widget class="QDialog" name="PipFileSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select File</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="fileLabel">
     <property name="text">
      <string>File Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="E5PathPicker" name="filePicker" native="true">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="userCheckBox">
     <property name="toolTip">
      <string>Select to install to the Python user install directory</string>
     </property>
     <property name="text">
      <string>Install into User Directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PipFileSelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PipFileSelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
